mappings/net/minecraft/command/EntitySelector.mapping

```diff
@@ -0,0 +1,22 @@
+CLASS dz net/minecraft/command/EntitySelector
+	FIELD a limit I
+	FIELD b includesEntities Z
+	FIELD d basePredicate Ljava/util/function/Predicate;
+	FIELD e distance Lbf$c;
+	FIELD f positionOffset Ljava/util/function/Function;
+	FIELD g box Lckl;
+	FIELD h sorter Ljava/util/function/BiConsumer;
+	FIELD j playerName Ljava/lang/String;
+	FIELD k entityId Ljava/util/UUID;
+	FIELD l type Ljava/lang/Class;
+	FIELD m checkPermissions Z
+	METHOD a getLimit ()I
+	METHOD a getEntity (Lca;)Lagi;
+	METHOD a getPositionPredicate (Lckp;)Ljava/util/function/Predicate;
+	METHOD a getEntities (Lckp;Ljava/util/List;)Ljava/util/List;
+	METHOD a getNames (Ljava/util/List;)Lja;
+	METHOD b getIncludesEntities ()Z
```

Comment:
We try to use is/should/can for booleans.

Reply:
Is `doesIncludeEntities` OK? I had a fiddle with `has` and `is`, but wasn't sure either worked.

Reply:
What about simply includesEntities?